Notes.md

```diff
@@ -0,0 +1,20 @@
+Goal: Let landlords create reviews for their tenants and be able to track how good/bad these tenants are so other landlords can decide whether or not to lease the places to them
+
+Landlords
+    -The primary user model
+    -2 separate show pages ( 1 for the profile page, 1 for the list of tenants)
```

Comment:
the 2nd show page is supposed to be a tenants show page, index page is different for the list of tenants

Reply:
Gotcha well can you correct it? I can't comment on the PR.